src/NuGet.Core/NuGet.Packaging/Signing/SignatureTrustResult.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// Represents a signature check result and any additional information
+    /// needed to display to the user.
+    /// </summary>
+    public class SignatureTrustResult
```

Comment:
What is the purpose of this class? Current implementation is simply a wrapper around the `SignatureTrust`.
Consider adding evaluation against the user's trust policy. e.g. - `SignVerifierSettings`.